Frameworks/Foundation/NSCFString.mm

```diff
@@ -89,12 +93,12 @@ - (instancetype)initWithFormat:(NSString*)format locale:(id)locale arguments:(va
     CFStringRef str;
 
     if (locale == nil) {
-        str = CFStringCreateWithFormatAndArguments(nullptr, nullptr, static_cast<CFStringRef>(format), argList);
+        str = _CFStringCreateWithFormatAndArgumentsAux(nullptr, &_NSCFStringCopyDescription, nullptr, static_cast<CFStringRef>(format), argList);
```

Comment:
Apparently not.

---
In reply to: [71573534](https://github.com/Microsoft/WinObjC/pull/671#discussion_r71573534) [](ancestors = 71573534,71446639)